packages/framer-motion/src/gestures/PanSession.ts

```diff
@@ -198,12 +195,12 @@ export class PanSession {
         onMove && onMove(this.lastMoveEvent, info)
     }
 
-    private handlePointerMove = (event: AnyPointerEvent, info: EventInfo) => {
+    private handlePointerMove = (event: PointerEvent, info: EventInfo) => {
         this.lastMoveEvent = event
         this.lastMoveEventInfo = transformPoint(info, this.transformPagePoint)
 
         // Because Safari doesn't trigger mouseup events when it's above a `<select>`
-        if (isMouseEvent(event) && event.buttons === 0) {
+        if (event.isPrimary) {
```

Comment:
This comment is probably no longer true, and I wonder if we should keep the logic